test/spec/api/create-vis.spec.js

```diff
@@ -242,21 +242,21 @@ describe('src/api/create-vis', function () {
     it('should instantiate map', function (done) {
       this.vis = createVis('domId', fakeVizJSON, {});
 
-      _.defer(function () {
+      setTimeout(function () {
         expect(mapInstantiationRequestDone()).toEqual(true);
         done();
-      });
+      }, 25);
```

Comment:
I strongly recommend against using timeouts in tests (including `_.defer`), since they cause the tests to fail in-deterministically eventually. I'd recommend to find some other way to delay the assertion, e.g. jasmine.clock.